dix/req.c

```diff
@@ -1402,6 +1408,20 @@ static void dix_rop(struct m0_dix_req *req)
 	M0_LEAVE();
 }
 
+/** Checks if the given cas get reply has a newer version of the value */
+static bool dix_item_version_cmp(struct m0_dix_item *ditem,
```

Comment:
In motr *cmp() functions follow memcmp(3) convention, i.e., return +ve, 0 or +ve depending on the "sign" if comparison. There are also various *_eq() functions that return a Boolean.

Reply:
Fixed